### PR TITLE
[DVCSMP-3653] On-boarding Fibaro Wall Plug EU ZW5

### PR DESCRIPTION
**Pull request by sagar-trility**

### CODE REVIEW

File: devicetypes/fibargroup/fibaro-wall-plug-eu-zw5.src/fibaro-wall-plug-eu-zw5.groovy [hunk 1]
```diff
@@ -0,0 +1,379 @@
+/**
+ *  Fibaro Wall Plug ZW5
+ */
+metadata {
+    definition (name: "Fibaro Wall Plug EU ZW5", namespace: "FibarGroup", author: "Fibar Group") {
```
Comment (greens) [inline devicetypes/fibargroup/fibaro-wall-plug-eu-zw5.src/fibaro-wall-plug-eu-zw5.groovy]: add generic ui metadata

File: devicetypes/fibargroup/fibaro-wall-plug-eu-zw5.src/fibaro-wall-plug-eu-zw5.groovy [hunk 2]
```diff
@@ -0,0 +1,379 @@
...
+metadata {
+    definition (name: "Fibaro Wall Plug EU ZW5", namespace: "FibarGroup", author: "Fibar Group") {
+        capability "Switch"
+        capability "Energy Meter"
+        capability "Power Meter"
+        capability "Configuration"
+        capability "Health Check"
+
+        command "reset"
+        command "refresh"
```
Comment (greens) [inline devicetypes/fibargroup/fibaro-wall-plug-eu-zw5.src/fibaro-wall-plug-eu-zw5.groovy]: replace with "Refresh" capability

File: devicetypes/fibargroup/fibaro-wall-plug-eu-zw5.src/fibaro-wall-plug-eu-zw5.groovy [hunk 3]
```diff
@@ -0,0 +1,379 @@
...
+
+        fingerprint deviceId: "0x1001", inClusters:"0x5E,0x22,0x59,0x56,0x7A,0x32,0x71,0x73,0x98,0x31,0x85,0x70,0x72,0x5A,0x8E,0x25,0x86"
```
Comment (greens) [inline devicetypes/fibargroup/fibaro-wall-plug-eu-zw5.src/fibaro-wall-plug-eu-zw5.groovy]: this fingerprint is redundant due to the more general one below

File: devicetypes/fibargroup/fibaro-wall-plug-eu-zw5.src/fibaro-wall-plug-eu-zw5.groovy [hunk 4]
```diff
@@ -0,0 +1,379 @@
...
+            )
+        }
+
+        input ( name: "logging", title: "Logging", type: "boolean", required: false )
+    }
+}
+
+//UI and tile functions
+def on() {
+    encap(zwave.basicV1.basicSet(value: 255))
```
Comment (greens) [inline devicetypes/fibargroup/fibaro-wall-plug-eu-zw5.src/fibaro-wall-plug-eu-zw5.groovy]: we prefer to use hex values here

File: devicetypes/fibargroup/fibaro-wall-plug-eu-zw5.src/fibaro-wall-plug-eu-zw5.groovy [hunk 5]
```diff
@@ -0,0 +1,379 @@
...
+    refresh()
+}
+
+//Configuration and synchronization
+def updated() {
+    if ( state.lastUpdated && (now() - state.lastUpdated) < 500 ) return
+    def cmds = []
+    logging("Executing updated()","info")
+
+    if (device.currentValue("numberOfButtons") != 6) { sendEvent(name: "numberOfButtons", value: 6) }
```
Comment (greens) [inline devicetypes/fibargroup/fibaro-wall-plug-eu-zw5.src/fibaro-wall-plug-eu-zw5.groovy]: unused

**Review by greens — CHANGES_REQUESTED** [review]
also may want to preemptively convert spaces to tabs

**Comment by sagar-trility** [comment]
@greens Changes have been made. Please can you review the PR. Thank You.

**Review by greens — APPROVED** [review]

**Pull request merged**